source/hic_hal/stm32/stm32h743ii/adc.c

```diff
@@ -0,0 +1,105 @@
+#include "adc.h"
+#include "stm32h743xx.h"
+#include "stm32h7xx_hal.h"
+
+// Timeout value in millisecond
+#define ADC_CONVERSION_TIMEOUT (10)
```

Comment:
Can you make this ADC_CONVERSION_TIMEOUT_MS, so the unit is clear in the location where it is used?